### ISSUE TITLE
require-description-complete-sentence: false positive with Markdown table

### ISSUE THREAD

**Issue by sveyret** [gajus/eslint-plugin-jsdoc#423]
If I have a comment which contains a Markdown table:

```
/**
 * This method is working on 2 steps.
 *
 * | Step | Comment     |
 * |------|-------------|
 * |   1  | do it       |
 * |   2  | do it again |
 */
```

The rule `require-description-complete-sentence` requires/adds a period after the table.

**Comment by ExE-Boss** [comment]
What about tables like:

```markdown
Still |   a
----- | -----
valid | table
```

Still |   a
----- | -----
valid | table

**Comment by brettz9** [comment]
@ExE-Boss : Yeah, there are so many possibilities here. Unless we are going to install a Markdown processor and/or grammar processor, I don't think we can adequately address all cases for this rule.
